gems/aws-sdk-core/spec/aws/plugins/endpoint_discovery_spec.rb

```diff
@@ -228,7 +242,7 @@ module Plugins
           region: 'us-east-1',
           endpoint_discovery: true,
           active_endpoint_cache: true,
-          endpoint_cache_poll_interval: 1,
+          endpoint_cache_poll_interval: 0.01,
```

Comment:
Why change these?

Reply:
This test was sleeping for 5 seconds to give the 1 second cache poll time to run in a seperate thread.  Lowering this value and then decreasing sleep saves 4.9 seconds.  Its not required, just was annoying me that these specs were taking so long.